cmd/template/cluster/provider/aws.go

```diff
@@ -30,6 +32,10 @@ func WriteAWSTemplate(out io.Writer, config ClusterCRsConfig) error {
 		return microerror.Mask(err)
 	}
 
+	if config.ControlPlaneSubnet != "" {
+		crs.AWSCluster.Annotations[annotation.AWSSubnetSize] = config.ControlPlaneSubnet
+	}
+
```

Comment:
I was not sure where to add this logic but I did not find in apiextensions the way to configure it